app/connectors/AccreditationConnector.scala

```diff
@@ -35,7 +35,6 @@ class AccreditationConnector @Inject() (config: Configuration, httpClient: HttpC
   private val clientIdAndAcceptHeaders                         =
     Seq("X-Client-ID" -> "tgp-frontend", "Accept" -> "application/vnd.hmrc.1.0+json")
 
-  //TODO call should go to data store instead of router directly
```

Comment:
are we not doing this TODO? I feel it should go to the data-store

Reply:
It’s not needed right now.